lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -1792,7 +1792,7 @@ void ByteCodeGenerator::FinalizeRegisters(FuncInfo* funcInfo, Js::FunctionBody*
         }
     }
 
-    // NOTE: The FB expects the yield reg to be the final non-temp.
+    // NOTE: The FunctionBody expects the yield reg to be the final non-temp.
```

Comment:
Ride-along comment improvement as I was tracking through where different registers go it took me a while to work out what an FB was